app/assets/stylesheets/pages/_home.scss

```diff
@@ -4,6 +4,37 @@ body {
   background-color: white;
 }
 
-h1 {
-  color: black;
+
+.banner {
+  h1 {
+    color: black;
+  }
+  p {
+    color: black;
+  }
+}
+
+.primary-color {
+  background-color: $main-orange;
+  border: none;
+    &:hover {
```

Comment:
minor indentation: 
&:hover {
   background-color: $main-orange;
   border: none;
   &:active {
      background-color: $main-orange;
   }
}

Reply:
unhelpfully it has removed my spacings...